src/libcrun/container.c

```diff
@@ -769,6 +769,8 @@ wasmer_do_exec (void *container, void *arg, const char *pathname, char *const ar
   wasm_module_t *module;
   wasm_instance_t *instance;
   wasm_extern_vec_t exports;
+  size_t args_size = 0;
+  char *wasi_args;
```

Comment:
can this be `cleanup_free char *wasi_args = NULL;` ?

Reply:
sure we could do this.